pkg/compiler/builtin_test.go

```diff
@@ -105,3 +105,112 @@ func TestMaxMin(t *testing.T) {
 		}
 	}
 }
+
+func TestClear(t *testing.T) {
+	tests := []struct {
+		name        string
+		src         string
+		expectedErr string
+		expected    any
+	}{
+		{
+			name: "invalid type array",
+			src: `
+				package foo
+				func Main() [42]int {
+					var m = [42]int{}
```

Comment:
Just for the record: arrays are not supported yet, ref. https://github.com/nspcc-dev/neo-go/issues/3524.

But this test should be kept, it's a valid one.